themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl

```diff
@@ -26,7 +26,7 @@
   {block name='product_cover'}
     <div class="product-cover">
       {if $product.cover}
-        <img class="js-qv-product-cover" src="{$product.cover.bySize.large_default.url}" alt="{$product.cover.legend}" title="{$product.cover.legend}" style="width:100%;" itemprop="image">
+        <img class="js-qv-product-cover" src="{$product.cover.bySize.large_default.url}" alt="{($product.cover.legend != '') ? $product.cover.legend : $product.name}" title="{$product.cover.legend}" style="width:100%;" itemprop="image">
```

Comment:
Thank you for this PR. I totally agree that image `alt` enhance page SEO, but the question is : Do classic theme specs include such improvements ? Why not to recommend to merchant to fill the legend of cover image for better SEO instead of more code. If not, my suggestion is to avoid inline stuff, maybe better to do:
`{assign $imageAlt = $product.cover.legend != '' ? $product.cover.legend : $product.name }`
what do you think?